tests/Doctrine/Migrations/Tests/Metadata/Storage/TableMetadataStorageTest.php

```diff
@@ -98,6 +98,27 @@ public function testComplete() : void
         ], $rows);
     }
 
+    public function testCompleteWithFloatTime() : void
```

Comment:
Unfortunately this test is useless. The issue seems to be in how PDO handles the `Type::INTEGER` typehint for postgres (for SQLite seems fine). Do not know how to test it properly TBH.